extensions/amp-nexxtv-player/validator-amp-nexxtv-player.protoascii

```diff
@@ -42,22 +42,22 @@ tags: {  # <amp-nexxtv-player>
     value: "api"
     value: "static"
   }
-  attrs: {
-    name: "data-origin"
-    value_url: {
-      protocol: "https"
-      protocol: "http"
-      allow_empty: true
-    }
-  }
```

Comment:
@twifkak It's fine to remove `data-` attribute from validation since they're always valid, right?

Reply:
Yeah, that should be fine. All `data-` attributes are valid by default. This just means it'll no longer be required to contain an http(s) URL; I could contain anything. Since it's ignored in the new code, this shouldn't create any errors.

One question I have is if there are any pages in the wild that use a value other than https://embed.nexx.cloud/ right now. Hopefully there aren't and we can just make the change. But if there are, this seems like it might be a surprising change for them. If the origin is changed to https://embed.next.cloud, will the media still work? Will the site owners' other expectations still be met? (I'm not sure why people would choose a custom origin, so I don't want to guess what those expectations are.)

An alternative is to change the runtime to just show a placeholder if data-origin has been customized. I'm not sure what's better or worse for your users. Just wanted to point that out, make sure you had the chance to make the decision.

Sorry if this was already discussed elsewhere or if I'm overthinking things.

Reply:
This is a valid point.

@neko-fire Will documents that set `data-origin` continue to work properly?

Reply:
yes we made already sure that this will work as intended with https://embed.next.cloud